tests/db_functions/tests.py

```diff
@@ -671,3 +671,159 @@ def test_function_as_filter(self):
             Author.objects.exclude(alias=Upper(V('smithj'))),
             ['Rhonda'], lambda x: x.name
         )
+
+    def test_ord(self):
```

Comment:
Please put these tests in a separate file for each function (e.g. as done for `test_cast.py`).